docs/build/sdk/indexs.md

```diff
@@ -0,0 +1,222 @@
+# Indexs
+
+## getAllMarketIds
+
+You can use this function to get all market IDs in the Zeitgeiest.
+
+```typescript
+const sdk = await SDK.initialize(endpoint);
+
+const res = await sdk.models.getAllMarketIds();
+```
+
+[Code snippet](https://github.com/Whisker17/sdk-demo/tree/main/src/index/getAllMarketIds.ts)
+
+## getAllMarkets
+
+You can use this function to get all market in the Zeitgeiest.
+
+```typescript
+const sdk = await SDK.initialize(endpoint);
+
+const res = await sdk.models.getAllMarketIds();
+```
+
+[Code snippet](https://github.com/Whisker17/sdk-demo/tree/main/src/index/getAllMarkets.ts)
+
```

Comment:
I think we need to add `createCpmmMarketAndDeployAssets` here as well.

Reply:
> I think we need to add `createCpmmMarketAndDeployAssets` here as well.

You mean this?
![image](https://user-images.githubusercontent.com/12571049/169804200-465c94df-dc95-4a11-ac28-26f5ff0c75e5.png)

Reply:
Yes, the url redirects to nowhere since `createCpmmMarketAndDeployAssets` isn't available at `Indexs` page.

Reply:
> 

fixed it.

Reply:
> Yes, the url redirects to nowhere since `createCpmmMarketAndDeployAssets` isn't available at `Indexs` page.

fixed it.